light_painting/launch/t1_initialize.launch

```diff
@@ -4,12 +4,12 @@
 
     <!-- Launch Parameters -->
     <arg name="sim" default="true" />
-    <arg name="arduino_port" default="/dev/ttyACM0" />
+    <!-- <arg name="arduino_port" default="/dev/ttyACM0" /> -->
```

Comment:
Go ahead and remove this line entirely since the variable is no longer used by the launch file.